pkg/upgrade/upgrader/helper.go

```diff
@@ -59,6 +60,23 @@ func getOpenEBSVersion(d *appsv1.Deployment) (string, error) {
 	return d.Labels["openebs.io/version"], nil
 }
 
+func checkOpenEBSVersion(d *appsv1.Deployment) (string, error) {
```

Comment:
Should this function be called `isValidVersion`? Also there is a message in this function about replica.